paddle/fluid/pybind/pybind.cc

```diff
@@ -817,6 +817,10 @@ All parameter, weight, gradient are variables in Paddle.
           [](BuildStrategy &self, bool b) {
             self.enable_data_balance_ = b;
           })  // FIXME(chengudo): enable_data_balance seems not important
+      .def_property(
+          "multi_batch_merge_repeats",
```

Comment:
I suggest you example in test_pass_builder.py to insert a pass and avoid this API. Very few people use it.

Reply:
Tried, but failed, this pass must run before multi_device_graph_pass